src/widgets/_viewswitcher.scss

```diff
@@ -0,0 +1,92 @@
+viewswitcher button {
+    margin: 0;
+    padding: 0;
+    border-radius: 0;
+    border-top: 0;
+    border-bottom: 0;
+    box-shadow: none;
+    font-size: rem(13px);
+    -gtk-icon-palette: error @text_color, success @text_color, warning @text_color;
+
+    &:not(:checked):not(:hover) {
+        background: transparent;
+    }
+
+    &:not(:only-child):not(:last-child) {
+        border-right-width: 0;
+    }
+
+    &:not(only-child):first-child:not(:checked):not(:hover),
+    &:not(:checked):not(:hover) + &:not(:checked):not(:hover) {
+        border-left-color: transparent;
+    }
+
+    &:not(only-child):last-child:not(:checked):not(:hover) {
+        border-right-color: transparent;
+    }
+
+    &:not(:checked):hover:not(:backdrop) {
+        background-image: #{'image(lighter(@color_primary))'};
+    }
+
+    &:not(only-child):first-child:not(:checked):hover,
+    &:not(:checked):hover + &:not(:checked):not(:hover),
+    &:not(:checked):not(:hover) + &:not(:checked):hover {
+        border-left-color: #{'shade(@color_primary, 1.15)'};
+    }
+
+    &:not(only-child):last-child:not(:checked):hover {
+        border-right-color: #{'shade(@color_primary, 1.15)'};
+    }
+
+    &:not(:checked):hover:backdrop {
+        background-image: #{'image(@color_primary)'};
+    }
+
+    stack > box {
+
+        & label.active {
+            font-weight: bold;
+        }
+
+        &.narrow {
+            @extend .small-label;
+            padding-top: rem(5px);
+            padding-bottom: rem(5px);
+
+            & image,
+            & label {
+                padding-left: rem(8px);
+                padding-right: rem(8px);
+            }
+        }
+
+        &.wide {
+            padding: 0 9px;
+        }
+    }
+}
+
+headerbar viewswitcher button {
```

Comment:
You can actually nest this inside the block above as
```
headerbar & {
```

the `&` represent the parent selector